mongoose/v3/index.d.ts

```diff
@@ -312,7 +312,7 @@ declare module "mongoose" {
   export interface SchemaOption {
     autoIndex?: boolean;
     bufferCommands?: boolean;
-    capped?: boolean;
+    capped?: boolean|number|Object;
```

Comment:
object is not good. Please refer this type of { size: 1024, max: 1000, autoIndexId: true }.